src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Net.Sdk.WorkloadManifestReader
+{
+    public class SdkDirectoryWorkloadManifestProvider : IWorkloadManifestProvider
+    {
+        readonly string sdkRootPath;
+        readonly string sdkVersionBand;
+
+        public SdkDirectoryWorkloadManifestProvider(string sdkRootPath, string sdkVersionBand)
```

Comment:
@mhutch it is better to just input SDK version. And the SdkDirectoryWorkloadManifestProvider will convert that into a band, or the consumer need to understand SDK version scheme

Reply:
And a constructor taking `string sdkRootPath, string sdkVersionBand` makes it hard to dependency inject(i think that is the point for IWorkloadManifestProvider). Since getting sdkRootPath and sdkVersionBand is business logic. So you cannot inject a fully initialized SdkDirectoryWorkloadManifestProvider before in constructor of consuming class

Reply:
sdkRootPath -> dotnetRootPath. sdk root might be considered as `program files/dotnet/sdk`. We have DOTNET_ROOT env already, so that aligns with what we want

Reply:
That's fair. dotnetRootPath and sdkversion it is then.